common.sh

```diff
@@ -39,8 +39,12 @@ source $CONFIG
 # mirror images for installation in restricted network
 export MIRROR_IMAGES=${MIRROR_IMAGES:-}
 
+# The dev-scripts working directory
 WORKING_DIR=${WORKING_DIR:-"/opt/dev-scripts"}
 
+# The minimum amount of space required for a default installation, expressed in GB
+MIN_SPACE_REQUIRED=${MIN_SPACE_REQUIRED:=80}
```

Comment:
If this variable is only needed in sanitychecks.sh, perhaps just define it in there?

We have a large number of variables in common.sh already, so it would be good to keep only those variables which are really common defined here I think.

Reply:
Initially I defined in the `sanitychecks.sh` file, but then I noticed that all the settings were centralized in `commons.sh` and did not want to increase the level of configuration scattering, but agree that very likely it's better to keep only stuff that is in common.
Then remains open the problem about how to document it, since in the README file there's only  a generic mention to the settings contained in the `commons.sh`. Maybe we can a small additional configuration paragraph in the README for that.

Reply:
Personally I'd move the variable to `sanitychecks.sh` and mention the variable name in the error message, then it becomes self-documenting if anyone hits the validation error